src/transformers/configuration_utils.py

```diff
@@ -429,6 +429,22 @@ def torch_dtype(self, value):
         logger.warning_once("`torch_dtype` is deprecated! Use `dtype` instead!")
         self.dtype = value
 
+    @property
+    def rope_scaling(self):
+        logger.warning_once(
+            "`rope_scaling` has been deprecated. Please use `rope_parameters` instead, which contains all RoPE configuration "
```

Comment:
(pls set target version for removal, perhaps something far away in the future like v5.6?)